semver/parse_range.ts

```diff
@@ -50,9 +50,9 @@ function parseHyphenRange(range: string) {
   if (isWildcard(rightGroups.major)) {
     to = "";
   } else if (isWildcard(rightGroups.minor)) {
-    to = `<${+rightGroups.major + 1}.0.0`;
+    to = `<${+rightGroups.major! + 1}.0.0`;
```

Comment:
Can you please provide a justification for this non-null assertion? Ditto to other similar non-null assertions.

Reply:
Also, I was looking at [](https://github.com/denoland/deno_std/pull/4159/commits/db1f140075f6162047385ac4930b0ba2890e0c3d) as reference